layouts/partials/social.html

```diff
@@ -55,5 +55,12 @@
       <a class="glyph" alt="Twitter profile" href="https://twitter.com/{{ . }}"><img src="/icons/twitter.svg" height="24px" width="24px"></a>
      </div>
      {{ end }}
+    {{ if isset .Site.Params "reddituser" }}
```

Comment:
I took a different way to add a button. This was necessary to access the `.Site.BaseURL` object for the URL. `{{ with }}` rebases `.Site` an it is not accessible any more, according the docs.

I made the call to `.Site.BaseURL` because I think that absolute URL are a bit safer when the site is deployed to a URL like `https://www.example.org/user/`. A `<img src="/icons/example.svg" />` will then be looked up to `https://www.example.org/icons/example.svg` instead of `https://www.example.org/user/icons/example.svg` which might fail.

I am quite new to Hugo, as far as I know this is how URLs are handled or do I miss a magic feature of Hugo to rebase the relative URLs?

Reply:
Okay, I will change this to `{{ with }}`. Then everything is the same as the rest in the file.

Beside that, I will check if my everything works when the site is in a path like `https://www.example.org/user/` and open a new issue when this is true.

Reply:
Yes, please let me know! Things seem to work fine in the [demo site generated by the theme gallery](https://themes.gohugo.io/theme/indigo/), which hosts the site at `/theme/indigo`, but that may use some other tricks to make things show up properly.